docker-compose.yml

```diff
@@ -41,7 +40,9 @@ services:
       - "9181:9181"
       - "9444:9444"
```

Comment:
In keeper_config.xml/yaml RAFT protocol port is 9234 not important since the keeper is in standalone mode ...